sbom/examples/rpm/from-koji.py

```diff
@@ -7,10 +7,11 @@
 import sys
 from tempfile import TemporaryDirectory
 
+# Script requires these RPMs: brewkoji, rpmdevtools, rpm-build
+# Run with: ./from-koji.py brew <NVR>
+
 source_re = re.compile(r"^(Source\d+):\s*((.*/)?(.*))$")
-tarball_re = re.compile(
-    r"^([^0-9]*)-([0-9a-f\.-]*)[\.-][a-z]"
-)  # Obviously not universal
+tarball_re = re.compile(r"^([^0-9]*)-([0-9a-f\.-]*)[\.-][a-z]")  # Obviously not universal
```

Comment:
Looks like you (or your editor) reformatted this, but not in the style that `black` does?

Reply:
Ah, I have my `black` configured for 100-char line limit globally, so it changed this line while I saved the file. Do you want me to revert it?

Reply:
I'm going to add some linting/formatting harness in a separate PR so we get consistency in all scripts.

Reply:
Done in https://github.com/RedHatProductSecurity/security-data-guidelines/pull/16